scripts/daam_script.py

```diff
@@ -124,27 +138,33 @@ def run(self,
         # print(f"hijack_comments={prompt_analyzer.hijack_comments}, used_custom_terms={prompt_analyzer.used_custom_terms}")
         # print(f"fixes={prompt_analyzer.fixes}")
         
-        if any(item[0] in attentions for item in self.prompt_analyzer.used_custom_terms):
+        if any(item[0] in self.attentions for item in self.prompt_analyzer.used_custom_terms):
             print("Embedding heatmap cannot be shown.")
             
         global before_image_saved_handler
         before_image_saved_handler = lambda params : self.before_image_saved(params)
-                
-        with torch.no_grad():
```

Comment:
Since we're in a `process_batch`-function, we'll already be inside a `torch.no_grad()`-context.